prusti-contracts-impl/src/lib.rs

```diff
@@ -51,11 +51,24 @@ pub fn refine_trait_spec(_attr: TokenStream, tokens: TokenStream) -> TokenStream
 }
 
 #[proc_macro_attribute]
-pub fn extern_spec(_attr: TokenStream, tokens: TokenStream) -> TokenStream {
-    tokens
+pub fn extern_spec(_attr: TokenStream, _tokens: TokenStream) -> TokenStream {
+    let callsite_span = Span::call_site();
+    (quote_spanned!(callsite_span=> )).into()
```

Comment:
```suggestion
    TokenStream::new()
```
I don't think the span does anything when there are no tokens. Please also change `export_spec` and `prusti_use`.